proto/umee/incentive/v1/genesis.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive";
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "umee/incentive/v1/incentive.proto";
+
+// GenesisState defines the x/incentive module's genesis state.
+message GenesisState {
+  Params                    params               = 1 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram completed_programs   = 2 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram ongoing_programs     = 3 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram upcoming_programs    = 4 [(gogoproto.nullable) = false];
+  uint32                    next_program_id      = 5;
+  uint64                    last_rewards_time    = 6;
+  repeated cosmos.base.v1beta1.Coin total_bonded = 7 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  repeated BondAmount        bond_amounts        = 8 [(gogoproto.nullable) = false];
+  repeated PendingReward     pending_rewards     = 9 [(gogoproto.nullable) = false];
+  repeated RewardBasis       reward_bases        = 10 [(gogoproto.nullable) = false];
+  repeated RewardAccumulator reward_accumulators = 11 [(gogoproto.nullable) = false];
+  repeated Unbonding         unbondings          = 12 [(gogoproto.nullable) = false];
+}
+
+// BondAmount tracks the amount of coins of one uToken denomination bonded to a
+// given reward tier by a single account.
+message BondAmount {
+  string                   account = 1;
+  uint32                   tier    = 2;
+  cosmos.base.v1beta1.Coin amount  = 3 [(gogoproto.nullable) = false];
+}
+
+// PendingReward tracks the amount of rewards that a given account has calculated
+// but not yet claimed.
+message PendingReward {
+  string   account                                 = 1;
+  repeated cosmos.base.v1beta1.Coin pending_reward = 2 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+}
+
+// RewardBasis tracks the value of a given tier and lock denom's RewardAccumulator
+// at the last time a specific account calculated pending rewards for it.
+message RewardBasis {
```

Comment:
what's the difference between `RewardBasis` and `RewardAccumulator`? Can we rename `RewardBasis` to something more clear and update the comment?

Reply:
`RewardAccumulator` is global:
> `1u/ATOM` bonded to tier `3` at genesis would have accumulated `0.04ATOM and 0.01UMEE` incentive rewards. This increases every block when programs are ongoing.

`RewardBasis` is local:
> The last time account `umee1234...` bonded, began unbonding, was liquidated, or claimed rewards for `u/ATOM` at tier 3, the value of the `u/ATOM RewardAccumulator` was `0.03ATOM and 0.01UMEE`. Therefore it the user were to claim rewards now, their tier 3 bonded `u/ATOM` will give additional rewards (0.03 -> 0.04).
This is only modified when the user does one of the transactions listed above, which is required to keep reward calculations mathematically correct.

Reply:
So how about renaming `Basis` to `Local`?